### PR TITLE
release/0.1.0

### PR DESCRIPTION
**Pull request by mckunda**

### CODE REVIEW

**Comment by mckunda** [comment]
[[wav-revival-lib] v0.1.0](https://trello.com/c/gG3UMGgb/27-wav-revival-lib-v010)

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test_script:
+  - ctest -T Test
+  - ps: |
+      $XSLInputElement = New-Object System.Xml.Xsl.XslCompiledTransform
+      $XSLInputElement.Load("https://raw.githubusercontent.com/rpavlik/jenkins-ctest-plugin/master/ctest-to-junit.xsl")
+      $XSLInputElement.Transform((Resolve-Path .\Testing\**\Test.xml), (Join-Path (Resolve-Path .) "ctest-to-junit-results.xml"))
+      $wc = New-Object 'System.Net.WebClient'
+      $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\ctest-to-junit-results.xml))
+
```
Comment (serjihsklovski) [inline appveyor.yml:45]: add `deploy` section to GitHub Releases

**Review by serjihsklovski — CHANGES_REQUESTED** [review]
please fix `appveyor.yml`, section `deploy`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+notifications:
+  email: false
+
+branches:
+  except:
+    - ci/appveyor_integration
```
Comment (serjihsklovski) [inline .travis.yml]: remove

**Review by serjihsklovski — CHANGES_REQUESTED** [review]
fix `.travis,yml`

File: appveyor.yml (hunk 1 above)
Comment (mckunda) [reply appveyor.yml:45]: Fixed

File: .travis.yml (hunk 2 above)
Comment (mckunda) [reply .travis.yml]: Fixed

**Review by serjihsklovski — APPROVED** [review]
approved

**Pull request merged**